libraries/debian_handler.rb

```diff
@@ -111,17 +95,32 @@ def configure_from_unstable_channel(action_name)
         product_version: new_resource.version
       }
 
-      ENV['ARTIFACTORY_USERNAME'] = new_resource.artifactory_username
-      ENV['ARTIFACTORY_PASSWORD'] = new_resource.artifactory_password
+      # Set Artifactory creds when using unstable channel
+      if new_resource.channel == :unstable
+        converge_by("Use Artifactory username '#{new_resource.username}'") do
```

Comment:
Can we move this code into `installer()` in `helpers.rb` and use that to construct the installer here in order to reduce copy / pasta code?